implementations/lighthouse/database/migrations/2023_02_09_183019_create_users_table.php

```diff
@@ -4,18 +4,29 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-class CreateUsersTable extends Migration
+return new class extends Migration
 {
-    public function up(): void
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
```suggestion
    public function up(): void
```

Same with `down()` and the other migrations.